fluent-bit-docker-image.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: fluent-bit-docker-image
+  version: 20220322
+  epoch: 0
+  description: Docker image for Fluent Bit
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+vars:
+  FLB_DOCKER_BRANCH: "1.8"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/fluent/fluent-bit-docker-image
```

Comment:
what do we get when we install this package ? I don't understand the purpose of this package more than just adding a git repo.

Reply:
Yep it's strange for sure. If you look at https://github.com/wolfi-dev/os/blob/main/aws-for-fluent-bit.yaml#L58, I think this repo is just used for copying some configuration files to the working directory.

The issue with the current setup is that the auto-updates don't quite work, as the sha of the second git-checkout step in https://github.com/wolfi-dev/os/blob/main/aws-for-fluent-bit.yaml (and for two other packages in enterprise-packages) is overwritten on every package update. @ajayk suggested we decouple the two repos and make this one its own package, however minimal it is.

Reply:
Alright! I wasn't sure this was a good practice. That is why I asked about it.